src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/showMessage..st

```diff
@@ -0,0 +1,8 @@
+writing
+showMessage: aString
+	| charHex |
+	1 to: aString size do: [ :i | 
+		charHex := (aString at: i) asciiValue.
+		(aString at: i) = $\
```

Comment:
Is it copy from Python?
I guess $\ is supposed to be line separator which is "Character cr" in Pharo. Try following example:
```Smalltalk
('1234
67' at: 5) = Character cr
```